src/main/java/banking/Main.java

```diff
@@ -0,0 +1,17 @@
+package banking;
+
+import banking.configuration.DBManager;
+import banking.dao.AccountDAO;
+
+public class Main {
+    public static void main(String[] args) {
+        if (args.length == 2 && args[0].equals("-fileName")) {
+            DBManager connector = new DBManager(args[1]);
```

Comment:
Why the local variable is called `connector`?

Reply:
fixed